specification/schemas/Action.json

```diff
@@ -0,0 +1,33 @@
+{
+  "type": "object",
+  "required": [
+    "action_type",
+    "values"
+  ],
+  "additionalProperties": false,
+  "properties": {
+    "action_type": {
+      "type": "string",
+      "example": "update-query"
```

Comment:
`update-query` was removed from the proposal and is not something we will (initially) support.